### PR TITLE
Ground work for making mleap scala 2.13 compatible

### PR DESCRIPTION
**Pull request by dotbg**
1. Using sbt 1.9
2. Using most recent versions of the scala libraries where possible
3. Replaced scala-arm with more idiomatic scala 2.13+ way (scala-arm was not updated for a long time)
4. Updated scalatest.
5. Updated ClassloaderUtils to use java 9 compatible code.
6. Using java 9 code instead of the commons-io.

### CODE REVIEW

**Review by jsleight — COMMENTED** [review]
Wow thanks for the massive contribution!  I'm still working through it all (about half done), but wanted to at least let you know that I'm looking :)

Couple of tests failing still:

- TensorflowModelSpec saved model with
```
org.tensorflow.exceptions.TFFailedPreconditionException: Attempting to use uninitialized value Variable
```
- a spring test seems like it didn't compile correctly
```Error: Could not find or load main class @.etc.sbt.jvmopts
Caused by: java.lang.ClassNotFoundException: @.etc.sbt.jvmopts
```

**Comment by dotbg** [comment]
@jsleight  thanks a lot!
Unfortunately, Tensorflow is not published for my platform, so I debugging that test will take time :(

File: mleap-spark-testkit/src/main/scala/org/apache/spark/ml/parity/SparkParityBase.scala [hunk 1]
```diff
@@ -168,9 +171,11 @@ abstract class SparkParityBase extends FunSpec with BeforeAndAfterAll {
           case (actualElem: Double, expectedElem: Double) =>
             assert(actualElem ~== expectedElem relTol eps)
           case (actualElem: Float, expectedElem: Float) =>
-            assert(actualElem ~== expectedElem relTol eps)
+            assert(actualElem.doubleValue() ~== expectedElem.doubleValue() relTol eps)
```
[the remark this answers is not in the document]
Comment (dotbg) [reply mleap-spark-testkit/src/main/scala/org/apache/spark/ml/parity/SparkParityBase.scala]: indeed, fixed

**Comment by dotbg** [comment]
@jsleight thanks for the comments, I've finally made the build green

**Review by jsleight — APPROVED** [review]

**Pull request merged**